[2024-06-24T09:16:02.530+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-24T09:16:02.558+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: demo_dag_for_alerting_final_1.check_previous_run_status scheduled__2024-06-24T09:14:00+00:00 [queued]>
[2024-06-24T09:16:02.567+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: demo_dag_for_alerting_final_1.check_previous_run_status scheduled__2024-06-24T09:14:00+00:00 [queued]>
[2024-06-24T09:16:02.567+0000] {taskinstance.py:2306} INFO - Starting attempt 2 of 6
[2024-06-24T09:16:02.581+0000] {taskinstance.py:2330} INFO - Executing <Task(BranchPythonOperator): check_previous_run_status> on 2024-06-24 09:14:00+00:00
[2024-06-24T09:16:02.587+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1532) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-24T09:16:02.588+0000] {standard_task_runner.py:63} INFO - Started process 1533 to run task
[2024-06-24T09:16:02.589+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'demo_dag_for_alerting_final_1', 'check_previous_run_status', 'scheduled__2024-06-24T09:14:00+00:00', '--job-id', '3424', '--raw', '--subdir', 'DAGS_FOLDER/demo_dag_for_alerting.py', '--cfg-path', '/tmp/tmpm49qfdlf']
[2024-06-24T09:16:02.590+0000] {standard_task_runner.py:91} INFO - Job 3424: Subtask check_previous_run_status
[2024-06-24T09:16:02.636+0000] {task_command.py:426} INFO - Running <TaskInstance: demo_dag_for_alerting_final_1.check_previous_run_status scheduled__2024-06-24T09:14:00+00:00 [running]> on host bf895605831d
[2024-06-24T09:16:02.767+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='demo_dag_for_alerting_final_1' AIRFLOW_CTX_TASK_ID='check_previous_run_status' AIRFLOW_CTX_EXECUTION_DATE='2024-06-24T09:14:00+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-24T09:14:00+00:00'
[2024-06-24T09:16:02.768+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-24T09:16:02.786+0000] {baseoperator.py:399} WARNING - BranchPythonOperator.execute cannot be called outside TaskInstance!
[2024-06-24T09:16:02.791+0000] {helper_functions.py:76} ERROR - Error interacting with the database: name 'select_query' is not defined
[2024-06-24T09:16:02.791+0000] {python.py:237} INFO - Done. Returned value was: skip_run
[2024-06-24T09:16:02.792+0000] {branch.py:36} INFO - Branch into skip_run
[2024-06-24T09:16:02.792+0000] {skipmixin.py:178} INFO - Following branch skip_run
[2024-06-24T09:16:02.810+0000] {skipmixin.py:246} INFO - Skipping tasks [('start_run', -1)]
[2024-06-24T09:16:02.838+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-24T09:16:02.860+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=demo_dag_for_alerting_final_1, task_id=check_previous_run_status, run_id=scheduled__2024-06-24T09:14:00+00:00, execution_date=20240624T091400, start_date=20240624T091602, end_date=20240624T091602
[2024-06-24T09:16:02.883+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-24T09:16:02.926+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-06-24T09:16:02.931+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
